tests/unit_test/unit-test-s3.py

```diff
@@ -0,0 +1,58 @@
+import unittest
+import boto3
+from moto import mock_s3
+from config import region, s3_bucket, s3_bucket_tags
+
+
+class S3MockTests(unittest.TestCase):
+    @mock_s3
+    def test_create_bucket(self):
+        bucket_name = s3_bucket
+
+        s3_client = boto3.client("s3")
+        s3_client.create_bucket(
+            Bucket=bucket_name,
+            CreateBucketConfiguration={"LocationConstraint": region},
+        )
+        print("Bucket created successfully:", bucket_name)
+
+        response = s3_client.list_buckets()
+        buckets = response["Buckets"]
+        self.assertEqual(len(buckets), 1)
+        self.assertEqual(buckets[0]["Name"], bucket_name)
+
+    @mock_s3
+    def test_check_bucket_tags(self):
+        bucket_name = s3_bucket
+
+        s3_client = boto3.client("s3")
+        s3_client.create_bucket(
+            Bucket=bucket_name,
+            CreateBucketConfiguration={"LocationConstraint": region},
+        )
+
+        tags = s3_bucket_tags
+
+        tag_set = [{"Key": key, "Value": value} for key, value in tags.items()]
+        s3_client.put_bucket_tagging(
+            Bucket=bucket_name,
+            Tagging={"TagSet": tag_set},
+        )
+        response = s3_client.get_bucket_tagging(Bucket=bucket_name)
+        tags_res = response["TagSet"]
+        if tags:
+            print("Tags for bucket", bucket_name)
+            for tag in tags_res:
+                print(tag["Key"], ":", tag["Value"])
+        else:
+            print("No tags found for bucket:", bucket_name)
```

Comment:
please use logging instead of prints command

Reply:
Resolved in [d67f4bc](https://github.com/XgridInc/xc3/pull/18/commits/d67f4bcc757b95ad043c435e8c4acb4dd33c6a76)